packages/@ember/-internals/routing/lib/services/router.ts

```diff
@@ -10,7 +10,7 @@ import Route from '../system/route';
 import EmberRouter, { QueryParam } from '../system/router';
 import { extractRouteArgs, resemblesURL, shallowEqual } from '../utils';
 
-const ROUTER = symbol('ROUTER') as string;
+const ROUTER = (symbol('ROUTER') as unknown) as string;
```

Comment:
Following up on https://github.com/emberjs/ember.js/pull/19575#discussion_r671691460:

I get this TS error in my branch without it:

![image](https://user-images.githubusercontent.com/1325249/126145039-f61bdcec-f383-4d9d-b287-ac2a8027afe4.png)

This comes from the change in https://github.com/emberjs/ember.js/pull/19667/files#diff-8519bc32ded2c3feefab86f01e1308398bacc99f2c10d4b28d372c0853d33203. The type of the exported `symbol` changed there, as the return type of the function is now always a native `symbol` (was `string | symbol` before).